### DISCUSSION TITLE
How to effectively get the logs in the console?

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2776] [Q&A] [answered]
How to effectively get the logs in the console?

**Comment by mdmintz** [comment, marked as answer]
See the example script for it: [SeleniumBase/examples/test_console_logging.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_console_logging.py)

**Reply by Uno-count** [reply]
Thank you for your hard work Sir, what if the logs are already existed upon visiting the site, would that be applicable?

**Reply by mdmintz** [reply]
If the console logs aren't captured at the moment they are outputted to the console, then they aren't reachable anymore. Make sure you call `self.start_recording_console_logs()` before those logs are outputted to the console.
